apps/coscrad-frontend-e2e/src/e2e/ux-components/id-info-icon.e2e.cy.ts

```diff
@@ -0,0 +1,52 @@
+const termId = '9b1deb4d-3b7d-4bad-9bdd-2b0d7b110001';
+const compositeId = `term/${termId}`;
+
+describe('IdInfoIcon', () => {
+    beforeEach(() => {
+        cy.visit(`/Resources/Terms/${termId}`);
+    });
+
+    it('Should display the id icon for the TermDetailFullViewPresenter', () => {
+        cy.get(`[aria-label="Click to Copy ID ${termId}"]`).should('be.visible');
+    });
+
+    it('Should show the copy modal when clicked', () => {
+        cy.get(`[aria-label="Click to Copy ID ${termId}"]`).click();
+
+        cy.get('.MuiDialogContent-root').should('be.visible');
+    });
+
+    it('Should copy the resource ID when the clipboard button is clicked', () => {
+        cy.wrap(
+            Cypress.automation('remote:debugger:protocol', {
+                command: 'Browser.grantPermissions',
+                params: {
+                    permissions: ['clipboardReadWrite', 'clipboardSanitizedWrite'],
+                    origin: window.location.origin,
+                },
+            })
+        );
+
+        cy.window()
+            .its('navigator.permissions')
+            .then((permissions) => permissions.query({ name: 'clipboard-read' }))
+            .its('state')
+            .should('equal', 'granted');
+
+        cy.get(`[aria-label="Click to Copy ID ${termId}"]`).click();
+
+        cy.get('.MuiDialogContentText-root > :nth-child(1)').click();
+
+        cy.window()
+            .its('navigator.clipboard')
+            .then((clip) => clip.readText())
+            .should('equal', termId);
+
+        cy.get('.MuiDialogContentText-root > :nth-child(2)').click();
```

Comment:
I've looked into a couple of ways to make this a re-usable call rather than repeating twice, but I'm not sure what's best.  We can make the clipboard check a function, or it could be a command.  Any guidance welcome.